File: SDKs/Java/src/main/java/com/aspose/words/model/BookmarkData.java
package com.aspose.words.model;

public class BookmarkData {
  private String Name = null;
  private String Text = null;
  /**
	 * getName
	 * Gets String
	 * @return Name
	 */
  public String getName() {
    return Name;
  }

	/**
	 * setName
	 * Sets String
	 * @param Name String
	 */
  public void setName(String Name) {
    this.Name = Name;
  }

  /**
	 * getText
	 * Gets String
	 * @return Text
	 */
  public String getText() {
    return Text;
  }

	/**
	 * setText
	 * Sets String
	 * @param Text String
	 */
  public void setText(String Text) {
    this.Text = Text;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class BookmarkData {\n");
    sb.append("  Name: ").append(Name).append("\n");
    sb.append("  Text: ").append(Text).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words_Cloud_SDK_for_.NET/src/Com/Aspose/Words/Model/BookmarkData.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Words.Model {
  public class BookmarkData {
    public string Name { get; set; }

    public string Text { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class BookmarkData {\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Text: ").Append(Text).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
